serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/controller/ServiceRecordController.java

```diff
@@ -127,19 +128,19 @@ public ResponseEntity entity(
         serviceRecord.ifPresent(entity::setServiceRecord);
         entity.setInfoRecord(entityInfo.get());
         // TODO: temporary solution for multiple servicerecords
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)